packages/upstream/conf-perl.1/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+maintainer: "tim@gfxmonk.net"
+homepage: "https://www.perl.org/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+license: "GPL-1+"
+build: [["perl" "--version"]]
+depexts: [
+  [["debian"] ["perl"]]
```

Comment:
I trust that this is installed on xenserver, but I'd like to see a depexts stanza for `centos` as well before merging

Reply:
Done.